corehq/apps/data_analytics/tests/test_esaccessors.py

```diff
@@ -31,25 +31,20 @@ def setUp(self):
 
 
 @generate_cases([
-    ([
-         # differentiate by username
-         ('app', 'device', 'userid', 'username0', 1),
-         ('app', 'device', 'userid', 'username1', 2),
-     ],),
     ([
          # differentiate by userid
-         ('app', 'device', 'userid0', 'username', 1),
-         ('app', 'device', 'userid1', 'username', 2),
+         ('app', 'device', 'userid0', 1),
+         ('app', 'device', 'userid1', 2),
      ],),
     ([
          # differentiate by device
-         ('app', 'device0', 'userid', 'username', 1),
-         ('app', 'device1', 'userid', 'username', 2),
+         ('app', 'device0', 'userid', 1),
+         ('app', 'device1', 'userid', 2),
      ],),
     ([
          # differentiate by app
-         ('app0', 'device', 'userid', 'username', 1),
-         ('app1', 'device', 'userid', 'username', 2),
+         ('app0', 'device', 'userid', 1),
+         ('app1', 'device', 'userid', 2),
      ],),
 ], MaltAnalyticsTest)
 def test_app_submission_breakdown(self, combination_count_list):
```

Comment:
Was this the test suite where it was considered out-of-scope to add full tests? Notably, the code was modified to log when the user and user_id do not match, and that has no coverage here (I think?)

Reply:
Since I've removed the code that checks/logs the user/user_id mismatch, with the explanation for doing so found [here](https://github.com/dimagi/commcare-hq/pull/31001#discussion_r793001256), this is no longer a concern.